utils.js

```diff
@@ -0,0 +1,23 @@
+export function renderVillain(villain, playerHp) {
+    const newVillainEl = document.createElement('button');
+    const nameEl = document.createElement('p');
+    const emojiEl = document.createElement('p');
+    const hpEl = document.createElement('p');
+
+    nameEl.textContent = villain.name;
+    hpEl.textContent = villain.hp;
+
+    if (villain.hp > 0) {
+        emojiEl.textContent = '⭐';
+    } else {
+        emojiEl.textContent = '👻';
+    }
+    if (playerHp <= 0) {
```

Comment:
Im wondering if this is working...im still able to play the game after all players hit 0